services/ir/.ir.env

```diff
@@ -1,44 +1,4 @@
-NEOFS_IR_LOGGER_LEVEL=debug
-
-# Profiler section
-NEOFS_IR_PPROF_ENABLED=true
-## Server address. Empty value disables profiler
-NEOFS_IR_PPROF_ADDRESS=:6060
-## Timeout to shut down the HTTP server
-## Default: 30s
-#NEOFS_IR_PROFILER_SHUTDOWN_TIMEOUT=1m
-
-# Application metrics section
-NEOFS_IR_PROMETHEUS_ENABLED=true
-## Server address. Empty value disables metrics gathering
-NEOFS_IR_PROMETHEUS_ADDRESS=:9090
-## Timeout to shut down the HTTP server
-## Default: 30s
-#NEOFS_IR_METRICS_SHUTDOWN_TIMEOUT=1m
-
-# Toggling the sidechain-only mode
-NEOFS_IR_WITHOUT_MAINNET=false
-
-NEOFS_IR_MAINNET_ENDPOINT_CLIENT_0_ADDRESS=ws://main-chain:30333/ws
-NEOFS_IR_MORPH_ENDPOINT_CLIENT_0_ADDRESS=ws://morph-chain:30333/ws
-NEOFS_IR_TIMERS_EMIT=50
-NEOFS_IR_TIMERS_STOP_ESTIMATION_DIV=4
-NEOFS_IR_TIMERS_COLLECT_BASIC_INCOME_DIV=2
-NEOFS_IR_TIMERS_DISTRIBUTE_BASIC_INCOME_MUL=3
-NEOFS_IR_TIMERS_DISTRIBUTE_BASIC_INCOME_DIV=4
-
-NEOFS_IR_EMIT_STORAGE_AMOUNT=1000000000
-NEOFS_IR_NETMAP_CLEANER_ENABLED=true
-
 NEOFS_IR_CONTRACTS_NEOFS=f5dba2f67baa3cbb5595f0a7e46adf564ec9407a
-NEOFS_IR_CONTRACTS_PROCESSING=37d3232083dd2c1bed76bb28d1c992fc6262f699
-
-NEOFS_IR_MORPH_VALIDATORS=02b3622bf4017bdfe317c58aed5f4c753f206b7db896046fa7d774bbc4bf7f8dc2
-NEOFS_IR_AUDIT_PDP_MAX_SLEEP_INTERVAL=100ms
-
-NEOFS_IR_SETTLEMENT_BASIC_INCOME_RATE=100000000
-NEOFS_IR_SETTLEMENT_AUDIT_FEE=100000
-
-NEOFS_IR_LOCODE_DB_PATH=/locode/db
+NEOFS_IR_CONTRACTS_PROCESSING=332016219b9eb48348fb12694409dbefe29bdf30
```

Comment:
Why we save some parameters in `.env`?

Reply:
it is sad but some of our envs are [used](https://github.com/nspcc-dev/neofs-dev-env/blob/d8e825443ee73cc5fda38f83f11e8443a111ea1b/bin/deposit.sh#L21) in other places so simple removal breaks other places. i could not come up with a simple solution for that so decided to keep it as is and solve that in another PR (it needed)

Reply:
Then maybe we should not duplicate this parameters in [.yaml](https://github.com/carpawell/neofs-dev-env/blob/5841fed24ebcb583a5dc9a4f3d4814d38d1fe5f7/services/ir/cfg/config.yml#L66)?

Reply:
oh, yeah, that's a mistake, fixed